data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/LogstashMapper.java

```diff
@@ -17,26 +17,29 @@
  */
 public class LogstashMapper {
     public PipelineModel mapPipeline(LogstashConfiguration logstashConfiguration) {
-        LogstashPluginMapper pluginMapper = new DefaultPluginMapper();
-
-        List<PluginModel> sourcePluginModels = new LinkedList<>();
-        List<PluginModel> prepperPluginModels = new LinkedList<>();
-        List<PluginModel> sinkPluginModels = new LinkedList<>();
 
-        List<LogstashPlugin> inputLogstashPluginList = logstashConfiguration.getPluginSection(LogstashPluginType.INPUT);
-        if (inputLogstashPluginList != null)
-            inputLogstashPluginList.forEach(logstashPlugin -> sourcePluginModels.add(pluginMapper.mapPlugin(logstashPlugin)));
+        List<PluginModel> sourcePluginModels = mapPluginSection(logstashConfiguration, LogstashPluginType.INPUT);
+        PluginModel sourcePlugin = null;
         if (sourcePluginModels.size() > 1)
             throw new LogstashMappingException("More than 1 source plugins are not supported");
+        else if (sourcePluginModels.size() == 1)
+            sourcePlugin = sourcePluginModels.get(0);
+
+        List<PluginModel> prepperPluginModels = mapPluginSection(logstashConfiguration, LogstashPluginType.FILTER);
```

Comment:
We can rename this in a future PR. I'm updating the fields from `sourcePluginModel`, `prepperPluginModels`, `sinkPluginModels` to `source`, `preppers`, `sinks` so that the `JsonProperty` annotation matches the field names. Not a blocker for now.